src/tracing/index.ts

```diff
@@ -17,24 +17,20 @@ const AMPLITUDE_DUMMY_KEY = '00000000000000000000000000000000'
 export const STATSIG_DUMMY_KEY = 'client-0000000000000000000000000000000000000000000'
 
 Sentry.init({
+  // general configuration:
   dsn: process.env.REACT_APP_SENTRY_DSN,
   release: process.env.REACT_APP_GIT_COMMIT_HASH,
   environment: getEnvName(),
+  // exception reporting:
```

Comment:
I personally don't think we need this comment and the one on line `20`, looks pretty self explanatory? I would prefer the comment to provide additional context if necessary.

Reply:
I think these comments have value in that they keep the configurations separated. Despite Sentry implementing both exception reporting and performance tracing, I think it's easier to think of configuring them separately, so these comments encourage devs to keep these configurations grouped separately.